vertx-hazelcast/src/main/java/org/vertx/java/spi/cluster/impl/hazelcast/HazelcastAsyncMultiMap.java

```diff
@@ -69,14 +77,6 @@ public Void perform() {
             map.remove(entry.getKey(), v);
           }
         }
-        // remove directly from cache as well, b/c if the hazelcast does know the server id , it will never get thrown
```

Comment:
I feel a lot better with this going out again. This is what we added a version before, right?